.github/workflows/test.yaml

```diff
@@ -29,22 +35,46 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
 
+      - name: Get npm cache directory
+        id: npm-cache-dir
+        run: |
+          echo "::set-output name=dir::$(npm config get cache)"
+
       - name: Cache node modules
         id: cache-npm-ext
         uses: actions/cache@v3
         with:
-          path: node_modules
-          key: ${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}
+          path: ${{ steps.npm-cache-dir.outputs.dir }}
+          key: ${{ runner.os }}-ext-${{ matrix.node }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.OS }}-ext-${{ matrix.node }}-${{ env.cache-name }}-
+            ${{ runner.OS }}-ext-${{ matrix.node }}-
 
       - name: Install dependencies
+        env:
+          PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: 'true'
```

Comment:
Why is it needed? we need chromium, don't we?

Reply:
I've tried various fixes to make the tests work on CI, but none of them worked. Regarding this flag, I took it from here: https://github.com/mujo-code/puppeteer-headful